php/blocks/class-loader.php

```diff
@@ -191,22 +186,6 @@ public function get_block_attributes( $block ) {
 				unset( $attributes[ $field_name ]['default'] );
 				$attributes[ $field_name ]['items'] = array( 'type' => 'string' );
 			}
-
-			if ( ! empty( $field['source'] ) ) {
-				$attributes[ $field_name ]['source'] = $field['source'];
-			}
-
-			if ( ! empty( $field['meta'] ) ) {
-				$attributes[ $field_name ]['meta'] = $field['meta'];
-			}
-
-			if ( ! empty( $field['selector'] ) ) {
-				$attributes[ $field_name ]['selector'] = $field['selector'];
-			}
-
-			if ( ! empty( $field['query'] ) ) {
-				$attributes[ $field_name ]['query'] = $field['query'];
-			}
```

Comment:
These fields are literally never set, and don't actually do anything. Not sure how they ended up here.

Reply:
Yeah, interesting.